Framework/API/inc/MantidAPI/WorkspaceUnitValidator.h

```diff
@@ -33,7 +33,11 @@ namespace API {
 */
 class MANTID_API_DLL WorkspaceUnitValidator : public MatrixWorkspaceValidator {
```

Comment:
A more general design would have been to enhance the CompositeValidator, or make another one that allowed OR operations.